CHANGELOG.md

```diff
@@ -1,20 +1,24 @@
-# v0.17.0-RC1 (expected week of 2017-07-31)
+# v0.17.0-RC1 (2017-0-16)
 * Port `ChunkAggregator` to fs2
 * Add logging middleware
-* Add `request#withUri`, which updates `request#pathInfo`. Remove
-  `request#copy`, which could cause inconsistency with `request.pathInfo`.
 * Standardize on `ExecutionContext` over `Strategy` and `ExecutorService`
 * Implement `Age` header
 * Fix `Client#toHttpService` to not dispose until the body is consumed
-* Change `Content-Length.apply` to `Content-Length.unsafeFromLong`
 * Add a buffered implementation of `EntityDecoder[Multipart]`
-* Remove `name` from `AttributeKey`. `AttributeKey`s are compared by reference,
-  so the `name` was for display only.
-* Add `Uri#withFragment` and `Uri#withoutFragment`
 * In async-http-client, don't use `ReactiveStreamsBodyGenerator` unless there is
   a body to transmit. This fixes an `IllegalStateException: unexpected message
   type`
-* Upgrade to async-http-client-2.0.33
```

Comment:
I believe it got removed because it was instead upgraded to 2.0.34 but didn't get added to the upgrade section